cmd/wait/runner.go

```diff
@@ -199,5 +199,65 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		r.logger.LogCtx(ctx, "message", "CoreDNS is ready")
 	}
 
+	if r.flag.Provider != "KVM" {
```

Comment:
Feels like this should be lower-case maybe.

Reply:
I tried to look this up - will check again.